### PR TITLE
Update commands and sub-commands to use a standard print style

### PR DESCRIPTION
**Pull request by jamezp**
This PR mostly updates the commands to use a defined `print()` style in the `AbstractCommand`. This allows the colorized output to be disabled for environments like CI.

### CODE REVIEW

File: cli/src/main/java/org/wildfly/glow/cli/commands/AbstractCommand.java [hunk 1]
```diff
@@ -34,5 +39,85 @@ public abstract class AbstractCommand implements Callable<Integer> {
...
+
+    private PrintWriter stdout;
+
+    private PrintWriter stderr;
+    private CommandLine.Help.Ansi ansi;
+
+    @Override
+    public final Integer call() throws Exception {
+        // print main command usage
+        spec.commandLine().usage(System.out);
```
Comment (jfdenise) [inline cli/src/main/java/org/wildfly/glow/cli/commands/AbstractCommand.java:55]: @jamezp , the failing tests are caused by this line. We are not expecting any output when compact is enabled. More generally do we really want to print the usage each time a command is called?

Comment (jamezp) [reply cli/src/main/java/org/wildfly/glow/cli/commands/AbstractCommand.java:55]: Oops, I thought I ran the tests locally, but I must have done it before I made this change.

**Comment by jamezp** [comment]
I don't know what is going on with this conflict. I'm tempted to delete the commit as I'm definitely up to date with the main branch.

**Comment by jfdenise** [comment]
@jamezp , thank-you. that is great!

**Pull request merged**